src/renderers/shared/stack/reconciler/ReactCompositeComponent.js

```diff
@@ -362,9 +338,20 @@ var ReactCompositeComponentMixin = {
       markup = this.performInitialMount(renderedElement, hostParent, hostContainerInfo, transaction, context);
     }
 
+    var debugID = 0;
```

Comment:
It seems like debugID is always set with this._debugID. Is there a reason why you need a local variable for it in the first place, especially since it requires you to write some code that could be confusing like this?